cpp/cmake/fetch_rapids.cmake

```diff
@@ -11,6 +11,8 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+set(rapids-cmake-repo vuule/rapids-cmake)
+set(rapids-cmake-branch upgrade-nvcomp-3.0.0)
```

Comment:
After https://github.com/rapidsai/rapids-cmake/pull/451 is merged, revert these lines (before merging this PR).
```suggestion
```

Reply:
I forgot to commit this before merging. It's in https://github.com/rapidsai/kvikio/pull/316.